modules/ROOT/pages/advanced/security.adoc

```diff
@@ -127,7 +127,7 @@ Cross-Site Scripting (XSS) involves using website or application inputs to injec
 
 Although {productname} filters content such as scripts from the editor content, precautions should be taken to protect your users, such as enabling a Content Security Policy (CSP) and server-side filtering.
 
-For information on Cross-Site Scripting and how to reduce the risk of an attack, see: https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2017/Top_10-2017_A7-Cross-Site_Scripting_(XSS)[OWASP Top Ten 2017 -- Cross-Site Scripting (XSS)].
+For information on Cross-Site Scripting and how to reduce the risk of an attack, see: https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2017/Top_10-2017_A7-Cross-Site_Scripting_(XSS)[OWASP Top Ten 2017 _-_ Cross-Site Scripting (XSS)].
```

Comment:
This was correct previously as this is meant to be an em dash.

- [ ] Revert the changes to this file for `--`

Reply:
done